lib/swaps/SwapClientManager.ts

```diff
@@ -213,6 +226,11 @@ class SwapClientManager extends EventEmitter {
       initWalletPromises.push(keystorePromise);
     }
 
+    if (this.connextClient) {
+      const initWalletPromise = this.connextClient.initWallet(seedMnemonic);
```

Comment:
Just to be clear - this code is only called the very first time the user creates xud seed/wallet. If I'm reading the `ConnextClient` code correctly then it needs to call `initWallet` for every start. Is that correct?

If that's the case then we'll need to do changes our end to provide a seed for `ConnextClient`.

Reply:
We actually just discussed this internally. You probably explained this before, but could you remind us again:
a) which keys exactly are derived from this seed, if more than one
b) how is the key/keys used
c) why does it need a new seed on every start

Reply:
Am I correct in understanding that there'll be a mnemonic & seed associated with each channel in Connext? We definitely wouldn't want to use our "master" seed which is tied to on-chain funds as is currently being done.

Would it make sense to store connext channel seeds in the xud database and pass them to connext each time xud starts up?

Karl is right that the current implementation would not be able to initialize connext, as it would only call `initWallet` on the very first time we start xud.